jupyter_server/serverapp.py

```diff
@@ -553,16 +554,14 @@ class ServerApp(JupyterApp):
 
     This launches a Tornado-based Jupyter Server.""")
     examples = _examples
-    aliases = aliases
-    flags = flags
 
-    classes = [
-        KernelManager, Session, MappingKernelManager, KernelSpecManager,
-        ContentsManager, FileContentsManager, NotebookNotary,
-        GatewayKernelManager, GatewayKernelSpecManager, GatewaySessionManager, GatewayClient,
-    ]
     flags = Dict(flags)
     aliases = Dict(aliases)
+    classes = [
+            KernelManager, Session, MappingKernelManager, AsyncMappingKernelManager, KernelSpecManager,
```

Comment:
I just confirmed that traitlets associated with `MappingKernelManager` are also available (and active) when `AsyncMappingKernelManager` is configured!  So we can remove the AMKM entry (and import statement).  This should make @Zsailer happy! :smile: